hmcts/VH-Automation-Test
# Participant video off scenario

### Change description ###
Steps to test:
Create hearing for next 30 minutes with judge and one participant
Join hearing as a judge
Join hearing as participant
Press choose camera and microphone button
Toggle off join with your camera on/off
Start hearing as judge
Check that participant camera is turned off for judge and participant
